resources/qml/WelcomePages/WhatsNewContent.qml

```diff
@@ -21,39 +25,170 @@ Item
         anchors.top: parent.top
         anchors.horizontalCenter: parent.horizontalCenter
         horizontalAlignment: Text.AlignHCenter
-        text: catalog.i18nc("@label", "What's new in Ultimaker Cura")
+        text: catalog.i18nc("@label", "What's New")
         color: UM.Theme.getColor("primary_button")
         font: UM.Theme.getFont("huge")
         renderType: Text.NativeRendering
     }
 
-    Cura.ScrollableTextArea
+    Item
     {
-        id: whatsNewTextArea
-
+        id: topSpacer
         anchors.top: titleLabel.bottom
-        anchors.bottom: getStartedButton.top
-        anchors.topMargin: UM.Theme.getSize("wide_margin").height
-        anchors.bottomMargin: UM.Theme.getSize("wide_margin").height
-        anchors.left: parent.left
-        anchors.right: parent.right
+        height: UM.Theme.getSize("default_margin").height
+        width: UM.Theme.getSize("default_margin").width
+    }
+
+    Rectangle
+    {
+        anchors
+        {
+            top: topSpacer.bottom
+            bottom: whatsNewDots.top
+            left: parent.left
+            right: parent.right
+            horizontalCenter: parent.horizontalCenter
```

Comment:
Left and right anchors can't work at the same time with the horizontalCenter. In this case it makes more sence to remove the horizontalCenter, since the left and right ones also define the width of the rectangle.
```suggestion
```